src/content/docs/en/guides/build-with-ai.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Building Astro sites with AI tools
+sidebar:
+  label: Build with AI
+i18nReady: true
```

Comment:
```suggestion
tableOfContents:
  minHeadingLevel: 2
  maxHeadingLevel: 4
i18nReady: true
```
This would allow you to next all the AI tools as `<h4>`s.